README.md

```diff
@@ -910,6 +910,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Nishant Siddh](https://github.com/nishant-siddh)
   - [Nithin](https://github.com/Nithin108)
   - [Nitin Sharma](https://github.com/nitin-787)
+  - [Nitin Jerome](https://github.com/nitinrjn)
```

Comment:
Please rearrange your name according to alphabetical order. Add your name above `Nitin Sharma`

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
name added alphabetically